src/login/login.c

```diff
@@ -679,7 +679,7 @@ static void login_fromchar_parse_online_accounts(int fd, int id)
 	login->online_db->foreach(login->online_db, login->online_db_setoffline, id); //Set all chars from this char-server offline first
 	users = RFIFOW(fd,4);
```

Comment:
Probably change this to RFIFOL as well?
char.c has `WFIFOL(chr->login_fd,4) = i;`

Reply:
nice! I ended up converting it to struct format so we don't have to worry about those details